jsonapi_schema.json

```diff
@@ -11,21 +11,27 @@
                             "type": "string"
                         },
                         "duration": {
-                            "type": "number"
+                            "type": "integer",
+                            "min": 0,
+                            "max": 16777215
                         },
                         "gameMode": {
-                            "type": "string"
+                            "type": "string",
+                            "enum": ["casual", "ranked", "blitz_pvp_ranked", "casual_aral", "private", "private_party_draft_match", "private_party_blitz_match", "private_party_aral_match"]
```

Comment:
Our site gets matches via the game mode filter anyways, so at least for us I'm fine with being lax here.